### ISSUE TITLE
Supabase - Google Auth option is missing.

### ISSUE THREAD

**Issue by glnmahsun** [CallumBoase/plasmic-supabase#32]
since the supabase removed native smtp support recently, users will start to look out for an easier option like google auth which is pretty quick to set up. and this package should also have abilities to support most populer auth providers from supabase

**Comment by CallumBoase** [comment]
Agreed - that would be a great enhancement. It's not highest priority for us at this stage because we're still ironing out some issues with basic functionality, but definitely something I'd be keen to add in future. Thanks @glnmahsun
